web/discord_relay/queue.go

```diff
@@ -171,8 +181,64 @@ func sendPvpStatusChange(discord *discordgo.Session, event EventStruct) {
 	sendEvent(discord, event, event.Data.Content, color, emoji)
 }
 
+func sendVoiceText(discord *discordgo.Session, event EventStruct, voiceSessions *cache.Cache) {
+	transcript := event.Data.Content
+	if transcript == "" {
+		transcript = "..."
+	}
+
+	steamId := event.Data.SteamId
+	messageID, found := voiceSessions.Get(steamId)
+
+	if found == false {
+		log.Println("Creating new message for voice")
+		params := &discordgo.WebhookParams{
+			AllowedMentions: &discordgo.MessageAllowedMentions{
+				Parse: []discordgo.AllowedMentionType{},
+			},
+			Username:  event.Data.SteamName,
+			AvatarURL: event.Data.Avatar,
+			Embeds: []*discordgo.MessageEmbed{
+				{
+					Description: fmt.Sprintf("%v %v", EMOJI_VOICE, transcript),
+					Color:       COLOR_BLUE,
+				},
+			},
+		}
+		message, err := discord.WebhookExecute(WebhookId, WebhookSecret, true, params)
+
+		if err != nil {
+			log.Println(err)
+		}
+
+		voiceSessions.Set(steamId, message.ID, cache.DefaultExpiration)
+	} else {
+		log.Println("Updating existing message for voice")
+		params := &discordgo.WebhookEdit{
+			Embeds: []*discordgo.MessageEmbed{
+				{
+					Description: fmt.Sprintf("%v %v", EMOJI_VOICE, transcript),
+					Color:       COLOR_BLUE,
+				},
+			},
+		}
```

Comment:
you could just contruct the embed in once place instead of for both found and not found cases